game/player.py

```diff
@@ -1,20 +1,23 @@
 #!/usr/bin/env python
-from abc import ABCMeta, abstractmethod
-from game.gameboard import GameBoard
+from game.strategy_one import StrategyOne
 
 class Player(object):
-    __metaclass__ = ABCMeta
 
-    def __init__(self, color):
+    def __init__(self, color, strategy=None):
+        if strategy:
+            self._strategy = strategy
+        else:
+            self._strategy = StrategyOne()
         self._color = color.lower()
 
     def color(self):
         return self._color
 
+    def set_strategy(self, new_strategy):
```

Comment:
Some issues with this one:
- What is the point of having a setter if you are going to be assigning the inst var directly in the `__init__` method?
- You added this method but it seems that you are not calling it from anywhere.
- This is a new public method but there are no tests for it.